superset-frontend/src/dashboard/components/ColorSchemeControlWrapper.jsx

```diff
@@ -30,7 +30,6 @@ const propTypes = {
 
 const defaultProps = {
   colorScheme: undefined,
-  onChange: () => {},
```

Comment:
this one is making me nervous, can we remove the `.isRequired` instead?

Reply:
👍